include/pstl/internal/pstl_config.h

```diff
@@ -66,6 +66,9 @@
 #if (_OPENMP >= 201307) || (__INTEL_COMPILER >= 1600) || (__PSTL_GCC_VERSION >= 40900)
 #define __PSTL_PRAGMA_SIMD __PSTL_PRAGMA(omp simd)
 #define __PSTL_PRAGMA_SIMD_REDUCTION(PRM) __PSTL_PRAGMA(omp simd reduction(PRM))
+#elif defined(__clang__)
+#define __PSTL_PRAGMA_SIMD __PSTL_PRAGMA(clang loop vectorize(enable))
```

Comment:
According to the [LLVM documentation](https://www.llvm.org/docs/Vectorizers.html) this pragma is about auto-vectorization (not SIMD vectorization)

> The `#pragma clang loop` directive **allows** loop vectorization hints...

As far as I know it's not similar to what `#pragma omp simd` does. At least Intel Compiler has the same difference.
Is it correct for Clang? Maybe Clang has only auto-vectorization...

Reply:
Clang does not support `__Pragma (simd)` so i thought to change it in this way, if that's not what we want we can just do `#define __PSTL_PRAGMA_SIMD` for Clang